pilot/pkg/model/jwks_resolver.go

```diff
@@ -78,7 +84,8 @@ type jwksResolver struct {
 	// map key is jwksURI, map value is jwtPubKeyEntry.
 	keyEntries sync.Map
 
-	client        *http.Client
+	httpsClient   *http.Client
```

Comment:
[nit] use httpClient and secureHttpClient or something like that. httpClient and httpsClient is too much similar and may cause bug due to mistyping.

Reply:
done